.github/workflows/pr-gpu.yaml

```diff
@@ -12,14 +12,9 @@ jobs:
     strategy:
       matrix:
         include:
-          - name: 'gpu-3.10-1.13-vision-doctest'
-            container: mosaicml/pytorch_vision:1.13.1_cu117-python3.10-ubuntu20.04
-            markers: 'not daily and not remote and gpu and (vision or not vision) and (doctest or not doctest)'
-            pytest_command: 'coverage run -m pytest'
-            composer_package_name: 'mosaicml'
-          - name: 'gpu-3.10-2.0-vision-doctest'
-            container: mosaicml/pytorch_vision:2.0.1_cu117-python3.10-ubuntu20.04
-            markers: 'not daily and not remote and gpu and (vision or not vision) and (doctest or not doctest)'
+          - name: 'gpu-3.10-2.1'
```

Comment:
are we not testing 2.0 and 1.3 images on torch anymore? I guess no need since we are migrating to training with more recent torch versions?

Reply:
+1 we shouldn't drop 1.13 and 2.0 tests yet, right?

Reply:
This is on the PR tests. The daily tests will still test all versions.